e2e/app/context.go

```diff
@@ -0,0 +1,56 @@
+// SPDX-FileCopyrightText: The RamenDR authors
+// SPDX-License-Identifier: Apache-2.0
+
+package app
+
+import (
+	"context"
+	"time"
+
+	"go.uber.org/zap"
+
+	"github.com/ramendr/ramen/e2e/config"
+	"github.com/ramendr/ramen/e2e/types"
+)
+
+// Context implements types.Context for sharing the log, env, and config with all code.
+type Context struct {
+	log    *zap.SugaredLogger
+	env    *types.Env
+	config *config.Config
+	ctx    context.Context
+}
+
+func NewContext(ctx context.Context, config *config.Config, env *types.Env, log *zap.SugaredLogger) Context {
+	return Context{
+		log:    log,
+		env:    env,
+		config: config,
+		ctx:    ctx,
+	}
```

Comment:
Why return Context and not *Context?

`test.NewContext()` accepts a `types.Context`, and `*Context` implements this interface.